### DISCUSSION TITLE
Getting blank screen on Release Mode

### DISCUSSION THREAD

**Discussion by SruthyJohn** [microsoft/react-native-windows#10741] [Q&A] [answered]
Hi Team,

We had developed a RN application long back and was forsaken. But recently had a situation where we need that project, and found out the dependency versions have moved upfront a lot. 

The steps we followed from creating the nodemodules to run in windows is as follows;

- npm install --legacy-peer-deps
- npx react-native-windows-init
- npx react-native autolink-windows --sln "windows\ivycpg_subd_reactnative.sln" --proj "windows\ivycpg_subd_reactnative\ivycpg_subd_reactnative.csproj"
- npx react-native run-windows --release

react dependencies in package.json is as follows;
`react": "18.0.0",
    "react-native": "0.69.5",
    "react-native-bcrypt": "^2.4.0",
    "react-native-calendars": "^1.1288.2",
    "react-native-device-info": "^10.0.2",
    "react-native-elements": "^3.4.2",
    "react-native-file-viewer": "^2.1.5",
    "react-native-fs": "^2.20.0",
    "react-native-gesture-handler": "^2.6.0",
    "react-native-linear-gradient": "^2.6.2",
    "react-native-maps": "^1.3.1",
    "react-native-modal-datetime-picker": "^13.1.2",
    "react-native-modal-selector": "^2.1.1",
    "react-native-popup-menu": "^0.16.0",
    "react-native-reanimated": "^2.10.0",
    "react-native-router-flux": "^4.3.1",
    "react-native-safe-area-context": "^4.3.3",
    "react-native-screens": "^3.17.0",
    "react-native-simple-dialogs": "^1.5.0",
    "react-native-simple-toast": "^1.1.4",
    "react-native-sqlite-storage": "^6.0.1",
    "react-native-vector-icons": "^9.2.0",
    "react-native-webview": "^11.23.1",
    "react-native-windows": "0.69.9",
    "react-navigation": "^4.4.4",
    "react-redux": "^8.0.2",
    "react-router-redux": "^4.0.8",`

In debug mode it is working fine but the same is not working in release mode. In release it gives a blank white screen.
Kindly help on resolving this.

**Comment by chrisglein** [comment, marked as answer]
There was a blank screen issue on 0.69.7. So should have been fixed by 0.69.9. 
In release mode you're not going to see any red box errors. You can enable "use developer settings" mode in the instance settings: https://microsoft.github.io/react-native-windows/docs/native-api/ReactInstanceSettings#usedevelopersupport
Hopefully that helps.

**Reply by chrisglein** [reply]
@SruthyJohn was that helpful?
